core/src/main/bin/questdb.sh

```diff
@@ -202,10 +202,10 @@ function start {
     if [ "${QDB_CONTAINER_MODE}" != "" ]; then
         ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt 2>&1
     elif [ "${QDB_DISABLE_HUP_HANDLER}" = "" ]; then
-        ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt 2>&1 &
+        ${JAVA} ${JAVA_OPTS} -Dhello-to-stderr -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt &
```

Comment:
Ent version rust code is liable to produce critical messages into stderr. The stderr to stdout redirect was deliberate and should be kept. I realise this throws spanner in the works.

may be @eugenels or @amunra have ideas how to trap rust OOM erros and logs them to STDOUT?